src/App.tsx

```diff
@@ -10,33 +11,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [id, setId] = useState(tabs[0].id);
+
```

Comment:
According to the task: `Save the selectedTab in the App (the first tab is selected by default);`

You should create variable `selectedTab` and use it below.